src/test/java/org/opensearch/knn/index/mapper/KNNVectorFieldMapperTests.java

```diff
@@ -146,7 +146,7 @@ public void testTypeParser_build_fromKnnMethodContext() throws IOException {
         // Check that knnMethodContext takes precedent over both model and legacy
         ModelDao modelDao = mock(ModelDao.class);
 
-        SpaceType spaceType = SpaceType.COSINESIMIL;
+        SpaceType spaceType = SpaceType.DEFAULT;
```

Comment:
Out of curiosity, why this change?

Reply:
@vamshin is it okay default engine doesnt support cosine?

Reply:
We could add this by taking a normalization and then setting to innerproduct. But this would take some effort.

Reply:
ah good catch! I think still we need to make faiss as default based on all the advanced capabilities like nested fields, efficient filters, disk optimized search, in future GPUs. In long term makes sense to build the normalization.

Reply:
@shatejas FAISS doesn't support cosine yet.